tests/test_losca_City_Planning.py

```diff
@@ -0,0 +1,88 @@
+from datetime import datetime
+from os.path import dirname, join
+
+import pytest
+from city_scrapers_core.utils import file_response
+from freezegun import freeze_time
+
+from city_scrapers.spiders.losca_City_Planning import LoscaCityPlanningSpider
+
+test_response = file_response(
+    join(dirname(__file__), "files", "losca_City_Planning.json"),
+    url="https://planning.lacity.gov/dcpapi/meetings/api/all/commissions/2024",
+)
+spider = LoscaCityPlanningSpider()
+
+freezer = freeze_time("2024-10-21")
+freezer.start()
+
+parsed_items = [item for item in spider.parse(test_response)]
+
+freezer.stop()
+
+
+def test_count():
+    assert len(parsed_items) == 35
+
+
+def test_title():
+    assert parsed_items[0]["title"] == "Zoning Administration"
+
+
+def test_description():
+    assert parsed_items[0]["description"] == "South Valley ZA Hearing Agenda"
+
+
+def test_start():
+    assert parsed_items[0]["start"] == datetime(2025, 1, 30, 0, 0)
+
+
+def test_end():
+    assert parsed_items[0]["end"] is None
+
+
+def test_time_notes():
+    assert parsed_items[0]["time_notes"] == ""
+
+
+def test_id():
+    assert (
+        parsed_items[0]["id"]
+        == "losca_City_Planning/202501300000/x/zoning_administration"
+    )
+
+
+def test_status():
+    assert parsed_items[0]["status"] == "tentative"
```

Comment:
<img src="https://github.com/user-attachments/assets/4397c7e9-ab10-4079-80e2-d23a6d4a453e" />
<img src="https://github.com/user-attachments/assets/c36252cb-4582-4fdd-b144-9092fc39aa60" />

I see some descriptions having "cancelled" in them. This should cause the meetings to have a status of cancelled. Adding some test cases for other status like cancelled and passed would be good.